<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>withdraw</class>
 <widget class="QWidget" name="withdraw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="withdraw20">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdraw40">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>40</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdraw50">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdraw100">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amount">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="muusumma">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Muu summa</string>
   </property>
  </widget>
  <widget class="QLabel" name="saldo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Saldo 0 â‚¬</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
